ReactAndroid/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'maven'
 
 apply plugin: 'de.undercouch.download'
+apply plugin: 'com.neenbedankt.android-apt'
```

Comment:
Tks, the master branch still works on 2.1.3 and android-apt works with 2.2, I will try annotationProcessor.

`    dependencies {
        classpath 'com.android.tools.build:gradle:2.1.3'
        classpath 'de.undercouch:gradle-download-task:2.0.0'`